src/app/Shared/Model/Client_Isolate.model.ts

```diff
@@ -65,7 +65,20 @@ export const MainFilterList: DbKey[] = [
     "federalState",
 ];
 
-export const subFiltersList: string[] = ["serovar", "species", "serotyp"];
+export const subFiltersList: string[] = [
```

Comment:
Are these a mix of matrix and microorganisms sub filters? If yes i would split them into two structures to keep them separate. You can use ``allSubFilters = [...matrixSubFilters, ...microSubFilters]`` if you need the merged list.

Reply:
Done.